CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2022-2024 Intel Corporation
+# Copyright (C) 2023-2024 Intel Corporation
```

Comment:
hmm...the first date should not be changed - we could have copied this file from somewhere else and the original creation date should not be changed, I guess...

Reply:
fixed by adding date exception